### PR TITLE
[Merged by Bors] - doc(RepresentationTheory): add some doc for three basic files

### PR DESCRIPTION
**Pull request by faenuccio**
Add some doc concerning Schur's lemma, orthogonality of characters and irreducible representations being defined as simple objects.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RepresentationTheory/Character.lean [hunk 1]
```diff
@@ -15,6 +15,15 @@ import Mathlib.RepresentationTheory.Invariants
 This file introduces characters of representation and proves basic lemmas about how characters
 behave under various operations on representations.
 
+A key result is the orthogonality of characters for irreducible representations of finite group
+over an algebraically closed field whose characteristic doesn't divide the order of the group. It
+is the therem `char_orthonormal`
+
+# Implementation notes
+
+Irreducible representations are implemented categorically, using the `Simple` class deined in
```
Comment (semorrison) [inline Mathlib/RepresentationTheory/Character.lean:24]:
```suggestion
Irreducible representations are implemented categorically, using the `Simple` class defined in
```

File: Mathlib/RepresentationTheory/FdRep.lean [hunk 2]
```diff
@@ -125,7 +129,8 @@ open scoped Classical
 -- deterministic timeout.
 instance : HasKernels (FdRep k G) := by infer_instance
 
--- Verify that Schur's lemma applies out of the box.
+/-- Schur's Lemma: the dimension of the `Hom`-space between two irreducible representation is `0` if
+they are not isomorphic, and `1` if they are. It applies out of the box. -/
```
Comment (semorrison) [inline Mathlib/RepresentationTheory/FdRep.lean:133]:
```suggestion
they are not isomorphic, and `1` if they are. -/
```

**Comment by semorrison** [comment]
bors d+

File: Mathlib/RepresentationTheory/Character.lean [hunk 3]
```diff
@@ -15,6 +15,15 @@ import Mathlib.RepresentationTheory.Invariants
 This file introduces characters of representation and proves basic lemmas about how characters
 behave under various operations on representations.
 
+A key result is the orthogonality of characters for irreducible representations of finite group
+over an algebraically closed field whose characteristic doesn't divide the order of the group. It
+is the therem `char_orthonormal`
```
Comment (grunweg) [inline Mathlib/RepresentationTheory/Character.lean:20]:
```suggestion
is the theorem `char_orthonormal`

**Pull request closed without merging**